### DISCUSSION TITLE
怎么改变stencil（侧边栏）的列数，官网里用layout属性，我不会用

### DISCUSSION THREAD

**Discussion by masterMo66** [antvis/X6#2897] [Q&A]
![stencilQuestion](https://user-images.githubusercontent.com/60913914/202379514-3ebf0f84-2681-4f0f-a0a6-6a4bb773b0bf.png)
![Snipaste_2022-11-17_14-50-01](https://user-images.githubusercontent.com/60913914/202379522-d1c1d695-a656-48dd-82b0-2d6eb776bbeb.png)

**Comment by nielinjie** [comment]
我认为layout和layoutOptions是二选一。
改变列数可以用layoutOptions:{columns :1}
